### ISSUE TITLE
Build on clang greater than 8?

### ISSUE THREAD

**Issue by mrwesleycrusher** [carla-simulator/carla#2779]
I am trying to build Carla 0.9.8 on arch linux, and I have clang 10 installed, but Carla requires clang 8.
Is there a reason clang 8 is required? Can it not be built on newer versions of llvm?

**Comment by nsubiron** [comment]
The server-side is statically linked to Unreal Engine, so it needs to compile with the same toolchain, llvm-8. The client-side can be built with any toolchain, but for consistency everything is built with clang-8.

But for instance in the [CppClient Example](https://github.com/carla-simulator/carla/tree/master/Examples/CppClient) we build the client-side C++ API with gcc.

**Comment by mrwesleycrusher** [comment]
Ahhhhhh I see. I built Unreal from source though without issue. Also, I am not sure if I need to submit a seperate issue for this, but the link to the unreal build documentation points to the official Unreal Engine Wiki, which no longer exists (Epic Games' git repo README has the same issue). The instructions I followed were located at https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Build/BatchFiles/Linux/README.md
